### PR TITLE
test: cleanup names.py and structure it + some fixes to onboarding tests

### LINKED ISSUE #6539: Clean up Names.py redudant objects
Remove all redundant objects from Names.py

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #6539

Remove all unused names and structure them by section. Also add objectName to a lot of them.

I also fixed a couple of onboarding tests.

There are still onboarding tests that fail for me, but they failed on master. I think we should finish fixing them in another PR.

In the future, we should enforce that names should have an objectName and also are in the right structure.

### CODE REVIEW

**Comment by jrainville** [comment]
@alaibe can you test please.

The community test will fail because the intro input is broken. The Chat test is also broken and fixed in another PR. The wallet test should all work and the onboarding not fully.

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py [hunk 1]
```diff
@@ -2,168 +2,60 @@
```
Comment (alaibe) [inline test/ui-test/testSuites/suite_status/shared/scripts/names.py:2]: As the title is clean names.py. Could you try to split this file and use import?

Maybe 1 file per section + 1 global?

**Review by alaibe — APPROVED** [review]

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py (hunk 1 above)
Comment (jrainville) [reply test/ui-test/testSuites/suite_status/shared/scripts/names.py:2]: I didn't know we could do that! On it

Comment (jrainville) [reply test/ui-test/testSuites/suite_status/shared/scripts/names.py:2]: Done

**Comment by jrainville** [comment]
@Khushboo-dev-cpp @igor-sirotin can you review ASAP, because this clean up is very important for other tests to stay clean. Also, the conflicts are gonna be super annoying haha

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM :)

**Pull request merged**